app/src/main/java/net/squanchy/support/system/CurrentTime.kt

```diff
@@ -1,10 +1,11 @@
 package net.squanchy.support.system
 
+import net.squanchy.BuildConfig
 import org.joda.time.LocalDateTime
 
 class CurrentTime {
 
-    fun currentTimestamp(): Long = System.currentTimeMillis()
+    private val mockTime = LocalDateTime.parse("2017-10-27T12:00:00.000")
```

Comment:
Do you have a nice way of doing that? If you use the debug flag, the "debug" value must be written somewhere in the codebase. Otherwise we can create a debug version (as in, use the debug flavor) of `CurrentTimeModule` and provide a different value.